dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/IdentifiableServiceImpl.java

```diff
@@ -293,6 +293,12 @@ protected void setDefaultSorting(PageRequest pageRequest) {
   @Override
   @Transactional(rollbackFor = {RuntimeException.class, IdentifiableServiceException.class})
   public I update(I identifiable) throws IdentifiableServiceException {
+    try {
+      urlAliasService.validate(identifiable.getLocalizedUrlAliases());
+    } catch (Exception e) {
+      throw new IdentifiableServiceException("Validation error: " + e);
+    }
```

Comment:
- sollte das nicht nach Zeile 306 gemacht werden? Denn die dort aufgerufene Methode verändert die `localizedUrlAliases` u.U. nochmal.
- im `save` (nach Z. 245) wäre es vllt. auch noch nötig
- damit die `ValidationException` ein `rollback` auslöst, müsste sie noch in der Annotation ergänzt werden

Reply:
done (Punkte 1+2), bzw. hat sich erledigt (Punkt 3)